k8s/config/overrides.go

```diff
@@ -98,25 +92,21 @@ const (
 	EnvVarRemoteRunnerMemUserDescription = "The mem limit and req for the remote runner"
 	EnvVarRemoteRunnerMemExample         = "1024Mi"
 
-	EnvVarEVMKeys                = "EVM_KEYS"
-	EnvVarEVMKeysUserDescription = "The keys used to connect to the evm"
-	EnvVarEVMKeysExample         = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-
 	EnvVarInternalDockerRepo            = "INTERNAL_DOCKER_REPO"
 	EnvVarInternalDockerRepoDescription = "Use internal docker repository for some images"
 	EnvVarInternalDockerRepoExample     = "public.ecr.aws"
 
-	EnvVarEVMUrls                = "EVM_URLS"
-	EnvVarEVMUrlsUserDescription = "The RPC URLs used to connect to the chain"
-	EnvVarEVMUrlsExample         = "wss://mainnet.infura.io/ws/v3/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-
-	EnvVarEVMHttpUrls                = "EVM_HTTP_URLS"
-	EnvVarEVMHttpUrlsUserDescription = "The HTTP RPC URLs used to connect to the chain"
-	EnvVarEVMHttpUrlsExample         = "https://mainnet.infura.io/v3/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-
 	EnvVarLocalCharts                = "LOCAL_CHARTS"
 	EnvVarLocalChartsUserDescription = "Use local charts from the CTF repository directly"
 	EnvVarLocalChartsExample         = "true"
+
+	EnvBase64ConfigOverride             = "BASE64_CONFIG_OVERRIDE"
+	EnvBase64ConfigOverriderDescription = "Base64-encoded TOML config (should contain at least chainlink image and version)"
+	EnvBase64ConfigOverrideExample      = "W0NoYWlubGlua0ltYWdlXQppbWFnZT0icHVibGljLmVjci5hd3MvY2hhaW5saW5rL2NoYWlubGluayIKdmVyc2lvbj0iMi43LjEtYXV0b21hdGlvbi0yMDIzMTEyNyIKCltBdXRvbWF0aW9uXQpbQXV0b21hdGlvbi5HZW5lcmFsXQpkdXJhdGlvbj0yMDAK"
+
+	EnvBase64NetworkConfig            = "BASE64_NETWORK_CONFIG"
+	EnvBase64NetworkConfigDescription = "Base64-encoded TOML network config with default RPC/WS endpoints and private keys"
+	EnvBase64NetworkConfigExample     = "Nope :-)"
```

Comment:
lol

Reply:
hihi I thought it wouldn't add to much value to add random base64ed TOML, but if you think it makes sense I can do it